tests/clients/test_dapr_grpc_client.py

```diff
@@ -200,8 +202,98 @@ def test_invoke_binding_no_create(self):
         self.assertEqual({}, resp.binding_metadata)
         self.assertEqual(0, len(resp.headers))
 
+    def test_parse_endpoint(self):
```

Comment:
Can we move this to `tests/conf/helpers_test.py` per my other comment? :)

Reply:
Done! Thanks :)